src/components/TopBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './TopBar.css';
+import logo from '../logoTopBar.png';
+
+class TopBar extends Component {
+  constructor() {
+    super();
+    // TODO: Replace these information by the one returned from the OpenID api
+    this.state = {
+      picture: 'https://avatars2.githubusercontent.com/u/1298871?s=40&v=4',
+      name: 'Thiago Coimbra Lemos',
+      username: 'tclemos',
```

Comment:
Would you mind using these values:

* picture: 'https://avatars1.githubusercontent.com/u/83798?s=40&v=4'
* name: 'huskyCI Dashboard'
* username: 'huskyCI'

Reply:
Done.